src/Avalonia.Controls/Button.cs

```diff
@@ -328,6 +323,11 @@ protected override void UpdateDataValidation(AvaloniaProperty property, BindingN
             }
         }
 
+        protected override void OnPointerCaptureLost(PointerCaptureLostEventArgs e)
+        {
+            IsPressed = false;
```

Comment:
This change here is a bit... random.

Reply:
I seem to have swapped the order of the `OnPointerCaptureLost`/`UpdateDataValidation` methods, I'm not sure though why git's diff is marking this as added, it's unchanged from diff on the left-hand-side.